frontend/catalyst/api_extensions/error_mitigation.py

```diff
@@ -30,8 +30,15 @@
 from catalyst.jax_primitives import zne_p
 
 
+def polynomial_extrapolation(degree):
+    """utility to generate polynomial fitting functions of arbitrary degree"""
+    return lambda scale_factors, results: qml.transforms.poly_extrapolate(
+        scale_factors, results, degree
```

Comment:
```suggestion
from functools import partial

def polynomial_extrapolation(degree):
    """utility to generate polynomial fitting functions of arbitrary degree"""
    return partial(qml.transforms.poly_extrapolate, degree=degree)
```

Reply:
oh... this was my first instinct too, but thought that you could only use `partial` for named arguments, not positional ones. TIL! I'll change this.